### DISCUSSION TITLE
Satellite images

### DISCUSSION THREAD

**Discussion by kjboa** [andreynovikov/trekarta#152] [Q&A] [answered]
First of, thank you for creating this app it now has completly replaced all my other map apps.
One Thing kind of confuses me though: Preview images on the playstore show multiple map layers including a satellite layer. I have done a little research but have found no way to actually download satellite maps for free. An issue here on github also mentioned trekarta plugins but i have no idea how to make them work with the app. So is there a way to get satellite imagery in a format trekarta supports without having to pay thousands of euros?

**Comment by andreynovikov** [comment, marked as answer]
Not it any out-of-the-box way. But there are means to accomplish that. Preview images show custom maps. These are user maps in raster mbtiles or rmaps format. There are some ready to use maps in that format on the internet. There are also at least two ways to make that maps by yourself. They are described in my personal blog:

1. <https://andreynovikov.info/sasplanet-howto.html>
2. <https://andreynovikov.info/maptiler-howto.html>

As for exactly satellite imagery there are several free for personal use sources. I will not state it definitely but it looks like you can legally use (cache) Bing and Maxar imagery for personal use.
